### DISCUSSION TITLE
[rmf_traffic_schedule_node]: Failed to correctly load participant registry: yaml-cpp: error at line 15, column 11: bad conversion

### DISCUSSION THREAD

**Discussion by jamietjx** [osrf/rmf_core#326] [Q&A]
I tried to ros2 run the traffic schedule node and got this error. I am trying to integrate rmf_core with the fleet_adapter_mir.
![Screenshot from 2021-06-15 10-25-43](https://user-images.githubusercontent.com/77659587/121983865-99a5dc00-cdc4-11eb-88a5-48fdf1384c83.png)

**Comment by Yadunund** [comment]
This repository is archived. We have migrated our development to a new [organization page](https://github.com/open-rmf/rmf). Please follow the instructions there to install the latest RMF packages. If you still encounter a problem, please open a ticket there.

**Comment by mxgrey** [comment]
First, this `osrf/rmf_core` repo is considered deprecated, so I recommend shifting your attention to the new `open-rmf` organization: https://github.com/open-rmf/rmf. The recommended place for general discussions on RMF is [here](https://github.com/open-rmf/rmf/discussions).

The issue you're reporting was a known issue in an older version of RMF: https://github.com/open-rmf/rmf_ros2/issues/20

It should be fixed now by [this PR](https://github.com/open-rmf/rmf_ros2/pull/22), but you'll need to migrate to the `open-rmf` repos to get the fix.

In the meantime, there are some steps you could take to help us confirm that the latest version of RMF is fixed.
1. Look for a file named `.rmf_schedule_node.yaml` in the directory that you launch RMF from
2. [Create a gist](https://gist.github.com/)
3. Copy/paste the contents of `.rmf_schedule_node.yaml` into the gist
4. Save the gist and send us the link to it
5. Delete the file `.rmf_schedule_node.yaml` on your own computer

After that, building and running the latest version of RMF should prevent this issue from happening again.

**Reply by jamietjx** [reply]
Thank you so much for your help.

**Comment by jamietjx** [comment]
The problem is solved after deleting the file .rmf_schedule_node.yaml. Here is the link to the gist:https://gist.github.com/jamietjx/ad390d81d357bddd195c4cf26cdb4df5
[https://github.githubassets.com/images/modules/gists/gist-og-image.png]<https://gist.github.com/jamietjx/ad390d81d357bddd195c4cf26cdb4df5>
[FATAL] [1623770339.998705264] [rmf_traffic_schedule_node]: Failed to correctly load participant registry: yaml-cpp: error at line 59, column 11: bad conversion<https://gist.github.com/jamietjx/ad390d81d357bddd195c4cf26cdb4df5>
[FATAL] [1623770339.998705264] [rmf_traffic_schedule_node]: Failed to correctly load participant registry: yaml-cpp: error at line 59, column 11: bad conversion - .rmf_schedule_node.yaml
gist.github.com

________________________________

**Reply by mxgrey** [reply]
Thanks for providing the file; that helps me confirm my belief that this issue is fixed in the latest version.

Note that if you only delete the `.rmf_schedule_node.yaml` file, the problem will only be solved temporarily. The bug that corrupted that file will likely happen again unless you update to the latest version of RMF.
